akd/agents/aspect_search/prompts.py

```diff
@@ -0,0 +1,97 @@
+from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
+
+
+# =============================================================================
+# Retrieves closely related Wikipedia pages for a given topic, helping to 
+# identify relevant subjects and understand typical content.
+# =============================================================================
+
+gen_related_topics_inst = '''I'm writing a Wikipedia page for a topic mentioned below. Please identify and recommend some Wikipedia pages on closely related subjects. I'm looking for examples that provide insights into interesting aspects commonly associated with this topic, or examples that help me understand the typical content and structure included in Wikipedia pages for similar topics.
+Please list the urls in separate lines.
+
+Topic of interest: {topic}'''
+
+gen_related_topics_prompt = ChatPromptTemplate.from_template(gen_related_topics_inst)
+
+
+# =============================================================================
+# Generates a list of hypothetical Wikipedia editors, each representing a 
+# unique perspective to a topic, along with a description of their focus areas.
+# =============================================================================
+
+gen_perspectives_prompt_inst = '''You need to select a group of Wikipedia editors who will work together to create a comprehensive article on the topic. Each of them represents a different perspective, role, or affiliation related to this topic. You can use other Wikipedia pages of related topics for inspiration. For each editor, add description of what they will focus on.
+Give your answer in the following format: 1. short summary of editor 1:description\n2. short summary of editor 2: description\n...
+
+Wiki page outlines of related topics for inspiration:
+{examples}'''
+
```

Comment:
Can we edit/engineer this prompt geared towards science in general than general topic?